site/content/concepts/tenant-limits.md

```diff
@@ -0,0 +1,19 @@
++++
+title = "Limits"
+weight = 196
++++
+
+Eclipse Hono™ users can set the limits such as the maximum number of device connections allowed per tenant or the allowed data volume of the messages over a period of time per tenant. 
+
+Hono specifies an API `ResourceLimitChecks` that is used by the protocol adapters for the verification of the configured limits. A default implementation of this API is shipped with Hono. This default implementation uses the metrics data from the Prometheus back end to verify the limits if any configured. To enable and use the default Prometheus based limits implementation, please refer to the [Limits admin guide] ({{< relref "/admin-guide/limits-config.md" >}}). If needed, Hono users can bring their own implementation of the above API and use it. 
+
+## Connections Limit
+
+At the moment Hono supports only the limit on the device connections. Before accepting a new connection request from a device, the corresponding Protocol adapter checks if the number of existing connections for the given Tenant exceeds the configured limit or not. If exceeds, then the connection request is declined. 
+
+The MQTT and AMQP protocol adapters keep the connections longer opened than their counterparts such as HTTP. Thereby the MQTT and AMQP adapters are enabled to check the connection limits before accepting any new connection to a device.
+
+## Tenant Configuration
+The limits for a tenant can be set using the tenant configuration. Please refer to the [Tenant API]({{< relref "/api/Tenant-API.md#request-payload" >}}) for more details.
+
+
```

Comment:
IMHO this whole text should be incorporated into deployment/resource-limitation.md instead

Reply:
@sophokles73 IMHO it is good to provide links in the concept page on how to configure a _tenant_ with `resource-limits` and how to _enable_ and _configure_ the default Prometheus based implementation. FMPOV when someone takes a look at the concept, then this page is self-sufficient for him/her to configure and use it. In addition to it, it is also good to add the above text to the deployment/resource-limitation.md. WDYT?

Reply:
@sophokles73 Would you mind looking at the pushed changes :wink:.